js/src/cleantalk-comments-editscreen.js

```diff
@@ -5,8 +5,46 @@ function ct_is_ip(str){
 	return str.search(/^\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}$/);
 }
 
-jQuery(document).ready(function(){
+/**
+ * Shows a popup The Real Person when hovering over the cursor for Admin
+ * @param {string} id
+ */
+// eslint-disable-next-line no-unused-vars,require-jsdoc
+function apbctRealUserBadgeViewPopup(id) {
+    document.querySelectorAll('.apbct-admin-real-user-popup').forEach((el) => {
+        el.style.display = 'none';
+    });
+    let popup = document.getElementById(id);
+    if (popup != 'undefined') {
```

Comment:
if you compare with string - use typeof

`if (typeof popup !== 'undefined') `

If you compare with udenfined - remove the quotes

`if (popup !== undefined) `

Reply:
ok